DESCRIPTION

```diff
@@ -15,13 +15,15 @@ License: LGPL (>= 3)
 Depends:
     R (>= 3.0.0),
     crunch (>= 1.15.3),
-    dplyr
+    dplyr,
+    magrittr
```

Comment:
This is not necessary; (1) dplyr exports `%>%` too (https://github.com/tidyverse/dplyr/blob/master/NAMESPACE#L174) so it should already be available; (2) otherwise if you still want to importFrom, why not put it in Imports instead of Depends?